dds_web/api/project.py

```diff
@@ -746,7 +752,6 @@ class RemoveContents(flask_restful.Resource):
 
     @auth.login_required(role=["Unit Admin", "Unit Personnel"])
     @logging_bind_request
-    @dbsession
```

Comment:
This also handles any of the db errors in the `delete` endpoint,e.g. `project.current_status`, `project.has_been_available` etc, if we move this decorator from this enpoint, we also need to add a try except:

try: before `project = project_schemas.ProjectRequiredSchema [....]`
except, including everything caught in the decorator: after `if not project.files:`